### PR TITLE
add doc update to circleci job for base branch

### PR DESCRIPTION
**Pull request by stefandeml**

### CODE REVIEW

File: scripts/publish_book.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+apt-get update
+apt-get -qq install git python-minimal
+python --version
+cargo install mdbook
+cd zokrates_book && mdbook build
+git config --global user.email "stefan.deml+zokratesbot@decentriq.ch"
+git clone https://github.com/Zokrates/zokrates.github.io.git
+git clone https://github.com/davisp/ghp-import.git
+cd zokrates.github.io
+../ghp-import/ghp_import.py -n -p -f -m "Documentation upload" -b "master" -r https://zokratesbot:"$GH_TOKEN"@github.com/Zokrates/zokrates.github.io.git ../book
```
Comment (Schaeff) [inline scripts/publish_book.sh]: Shall we include the version number in the commit message?

File: scripts/publish_book.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Exit if any subcommand fails
+set -e
+
+apt-get update
+apt-get -qq install git python-minimal
+python --version
+cargo install mdbook
+cd zokrates_book && mdbook build
+git config --global user.email "stefan.deml+zokratesbot@decentriq.ch"
```
Comment (Schaeff) [inline scripts/publish_book.sh]: Is this required if we use a token?

Comment (stefandeml) [reply scripts/publish_book.sh]: yes, otherwise commits show up with "Unknown" as user.

File: scripts/publish_book.sh (hunk 1 above)
Comment (stefandeml) [reply scripts/publish_book.sh]: added tags to the commit msg (including a catch for now in case no tag exists)

File: scripts/publish_book.sh [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Exit if any subcommand fails
+set -e
+
+if [ "$CIRCLE_BRANCH" == "develop" ]; then
```
Comment (Schaeff) [inline scripts/publish_book.sh]: @stefandeml think we agreed for this to be master? Otherwise looks good!

Comment (stefandeml) [reply scripts/publish_book.sh]: As far as I see $CIRCLE_BRANCH is the source branch, hence we have to use "develop", no?

File: scripts/publish_book.sh [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    apt-get -qq install git python-minimal
+    python --version
+    cargo install mdbook
+    cd zokrates_book && mdbook build
+    git config --global user.email "stefan.deml+zokratesbot@decentriq.ch"
+    git clone https://github.com/Zokrates/zokrates.github.io.git
+    git clone https://github.com/davisp/ghp-import.git
+    cd zokrates.github.io
+    { 
+        TAG=$(git describe --tags) && echo $TAG
```
[the remark this answers is not in the document]
Comment (stefandeml) [reply scripts/publish_book.sh]: Fixed to use cargo version number

File: scripts/publish_book.sh (hunk 3 above)
Comment (Schaeff) [reply scripts/publish_book.sh]: No I don't think so: after we've merged from a release branch to master, CircleCI builds master which should trigger deployment. Or am I missing something?

Comment (Schaeff) [reply scripts/publish_book.sh]:
```suggestion
if [ "$CIRCLE_BRANCH" == "master" ]; then
```

**Pull request merged**